stored_requests/backends/http_fetcher/fetcher_test.go

```diff
@@ -61,14 +63,39 @@ func TestReqsAndImps(t *testing.T) {
 	assertErrLength(t, errs, 0)
 }
 
-func TestMissingValues(t *testing.T) {
```

Comment:
Curious, what's the reason for removing this missing value test case?

Reply:
accident, will add back